src/test/unit_tests/test_nc_nsfs_cli.js

```diff
@@ -530,6 +531,33 @@ mocha.describe('manage_nsfs cli', function() {
             }
         });
 
+        mocha.it('cli account add - uid=0, gid!=0', async function() {
```

Comment:
I would change the description so it will be clear for others... (other tests as well).
```suggestion
        mocha.it('cli account add - uid is 0, gid is not 0', async function() {
```